components/02-elements/form/input/input--textarea.hbs

```diff
@@ -1,14 +1,14 @@
-<div class="input {{ class }}" {{{ attribute }}}>
+<div class="input {{ class }}" {{ attribute }}>
     <label class="input__label {{ label.class }}"
            for="textarea"
```

Comment:
here and in `input` please move `for` and `id` to config file, b/c it will be easier to use it in modules/views

Reply:
done

Reply:
done